### PR TITLE
NBs code patterns

### PR DESCRIPTION
**Pull request by cfascina**
## Impact and Context
Code patterns for developed NBs.

## Testing
- [ ] Unit test
- [ ] Deployed to dev env
- [x] Other (explain) 

## Reverting
- [ ] Contains Migration - _Do Not Revert_

### CODE REVIEW

**Review by liuverta — CHANGES_REQUESTED** [review]
Sorry to trouble you, Caio—I noticed that we could improve the model/endpoint names a little to make them more accurate with how we'd like users to name them.

File: deployment/huggingface/yolos-seq2seq.ipynb [hunk 1]
```diff
@@ -134,7 +135,18 @@
...
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "MODEL_NAME = 'Object Detection - Hugging Face (Example)'\n",
+        "VERSION = 'v0'\n",
+        "ENDPOINT_NAME = 'object-detection-hugging-face'"
```
[the remark this answers is not in the document]
Comment (cfascina) [reply deployment/huggingface/yolos-seq2seq.ipynb:149]: Done.

File: deployment/tfhub/object-detection-url.ipynb [hunk 2]
```diff
@@ -139,7 +140,18 @@
...
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "MODEL_NAME = 'Object Detection - TFHub (Example)'\n",
+        "VERSION = 'url'\n",
+        "ENDPOINT_NAME = 'object-detection-tfhub-url'"
```
[the remark this answers is not in the document]
Comment (cfascina) [reply deployment/tfhub/object-detection-url.ipynb:154]: Done.

File: deployment/tfhub/object-detection-base64.ipynb [hunk 3]
```diff
@@ -143,7 +144,18 @@
...
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "MODEL_NAME = 'Object Detection - TFHub (Example)'\n",
+        "VERSION = 'base64'\n",
+        "ENDPOINT_NAME = 'object-detection-tfhub-base64'"
```
[the remark this answers is not in the document]
Comment (cfascina) [reply deployment/tfhub/object-detection-base64.ipynb:158]: Done.

File: deployment/pytorch/torch-hub-yolov5.ipynb [hunk 4]
```diff
@@ -122,6 +119,17 @@
...
     },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "MODEL_NAME = 'Object Detection - PyTorch (Example)'\n",
+        "VERSION = 'v0'\n",
+        "ENDPOINT_NAME = 'object-detection-pytorch'"
```
[the remark this answers is not in the document]
Comment (cfascina) [reply deployment/pytorch/torch-hub-yolov5.ipynb]: Done.

**Comment by cfascina** [comment]
> Sorry to trouble you, Caio—I noticed that we could improve some of the model/endpoint names a little to make them more accurate with how we'd like users to name them.

It really makes more sense with these changes :+1:

**Review by liuverta — APPROVED** [review]

**Pull request merged**